doc/rados/configuration/ceph-conf.rst

```diff
@@ -277,6 +280,48 @@ surrounded by square brackets. For example,
 	debug ms = 10
 
 
+Config file option values
+-------------------------
+
+The values of configuration options is a string. If it is too long to
+fit in a single line, you can put a backslash (``\``) at the end of line
+as the line continuation marker, so the value of the option will be
+the string after ``=`` in current line combined with the string in the next
+line::
+
+  [global]
+  foo = long long ago\
```

Comment:
this can not be parsed by the pygments INI lex parser on jenkins builder. so i am using the plain code block without syntax highlighting.